curriculum/challenges/english/blocks/workshop-registration-form/60fab8367d35de04e5cb7929.md

```diff
@@ -82,7 +91,11 @@ assert.isNotTrue(/^\s+|\s+$/.test(innerContent));
       </fieldset>
       <fieldset></fieldset>
 --fcc-editable-region--
-      <label for="terms-and-conditions"><input id="terms-and-conditions" type="checkbox" required /></label>
+      <p>
+      Read our <a href="https://www.freecodecamp.org/news/terms-of-service/">terms and conditions</a>
+      </p>
+        <label for="terms-and-conditions">
+          <input id="terms-and-conditions" type="checkbox" required />I accept the terms and conditions</label>
```

Comment:
this is a general comment that you should keep in mind when contributing to a workshop:
this is not something you can do, you should never add in the seed code what should be written by the learner. The seed code that needs to be updated is the one in the next step, and all the steps until the end, and then the solution